server/app/src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
 io.delta.sharing.api.server.defaultMaxResults=10
-quarkus.banner.path=banner.txt
\ No newline at end of file
+quarkus.banner.path=banner.txt
+
+whitefox.server.authentication.enabled=${WHITEFOX_SERVER_AUTHENTICATION_ENABLED:-false}
```

Comment:
you don’t need to set the property to the environment variable, quarkus does that automatically :) so you can remove the next line and simplify this one